apps/frontend/src/pages/dashboard/revenue/index.vue

```diff
@@ -84,10 +84,16 @@
 					</NuxtLink>
 				</ButtonStyled>
 			</div>
-			<p v-if="blockedByTax" class="text-sm font-bold text-orange">
+			<p v-if="blockedByTax && !blockedByTin" class="text-sm font-bold text-orange">
 				You have withdrawn over $600 this year. To continue withdrawing, you must complete a tax
 				form.
 			</p>
+			<p v-if="blockedByTin">
+				<span class="text-sm font-bold text-orange">
+					Your TIN/SSN did not match the IRS records. Please
+					<a class="underline" href="https://support.modrinth.com">contact support</a>.
+				</span>
+			</p>
```

Comment:
This can be simplified slightly by putting the blockedByTin div above the blockedByTax div, and then making the blockedByTax div a v-else-if